GenerateToolingFeed/V4Format/V4FormatFeedEntryUpdater.cs

```diff
@@ -12,7 +12,10 @@ internal class V4FormatFeedEntryUpdater : IFeedEntryUpdater
     {
         private readonly string _tag;
 
-        private readonly IDictionary<string, string> _dotnetToItemTemplates = new Dictionary<string, string>()
+        private const string _itemTemplatesKey = "itemTemplates";
```

Comment:
should `const` be pascal case? Even if private?


```suggestion
        private const string ItemTemplatesKey = "itemTemplates";
```

Reply:
Ah, you're right. I'll update that.